lib/utils/blockchain_explorer_utils.dart

```diff
@@ -0,0 +1,9 @@
+class BlockChainExplorerUtils {
+  String formatTransactionUrl({required String txid, required String mempoolInstance}) {
+    return "$mempoolInstance/tx/$txid";
+  }
+
+  String formatRecomendedFeesUrl({required String mempoolInstance}) {
```

Comment:
Typo in name
`formatRecommendedFeesUrl`

Reply:
Nice catch. Fixed